.github/workflows/testing.yml

```diff
@@ -0,0 +1,81 @@
+name: macOS tests
+
+on:
+  - push
+  - pull_request
+
+
+jobs:
+  test_macos:
+
+    runs-on: macos-latest
+    strategy:
+      max-parallel: 3
```

Comment:
should this be `2` now that py3.8 is removed?

Reply:
It doesn't matter, but I'm going to add 3.8 back in. It found #11827

Reply:
#11830 is in, feel free to add back in